outputs/build_summary.csv

```diff
@@ -1,4 +1,5 @@
 n_segment_cells,min_flow,max_to_route,time_total_mins,time_r_commute_mins,routing_date
+21975174,1,1000000000,2017.52,0,2023-10-07
```

Comment:
@mem48 we should update the results for the October build.

Reply:
See the data here: https://github.com/nptscot/outputdata/releases/download/v2023-10-18-08-46-09.827274_commit_57d2dcdd739f665bb8d3fb5540cb6bbd10fcb08f/combined_network_tile.zip